boards/arm/contextualelectronics_abc/doc/index.rst

```diff
@@ -0,0 +1,155 @@
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M)    | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| Modem     | on-board   | quectel_bg9x         |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `ABC Board website`_ for more details on this board, and
+`Nordic Semiconductor Infocenter`_ for a complete list of SoC
+features.
+
+Programming and Debugging
+*************************
+
+Applications for the ``contextualelectronics_abc`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the ABC board
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: contextualelectronics_abc
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+Overlay file naming
+===================
+
+The file has to be named ``<board>.overlay`` and placed in the app main directory to be
+picked up automatically by the build system.
```

Comment:
Looks like this is copy/pasted from elsewhere in the docs, but I don't think it should be here; it's just duplicating information in https://docs.zephyrproject.org/latest/guides/dts/howtos.html#set-devicetree-overlays (and slightly inaccurately, at that); please remove or replace with a link to that doc.

I've sent a separate PR  (https://github.com/zephyrproject-rtos/zephyr/pull/31123) fixing the others.